src/Core/System/SalesChannel/Context/BaseSalesChannelContextFactory.php

```diff
@@ -73,12 +77,20 @@ public function create(string $salesChannelId, array $options = []): BaseSalesCh
         $criteria->setTitle('base-context-factory::sales-channel');
         $criteria->addAssociation('currency');
         $criteria->addAssociation('domains');
+
+        $criteria->addAssociations([
```

Comment:
was this on purpose?

domains is added twice here, and both associations are already added in the two lines above 🤔

Reply:
forgot to remove it actually 😿